docs/websites/cms/install-and-configure-wordpress-using-wp-cli.md

```diff
@@ -0,0 +1,273 @@
+
+## Prerequisites
+Before we move ahead, make sure you have completed the following guides:
+
+* [Getting Started with Linode](/docs/getting-started)
+* [How to Install a LAMP Stack on Ubuntu 14.04](/docs/websites/lamp/how-to-install-a-lamp-stack-on-ubuntu-14-04)
+* [Securing your Server](/docs/security/securing-your-server)
+
+{: .note}
+>
+>This guide is written for a non-root user. Commands that require elevated privileges are prefixed with `sudo`. If you're not familiar with the `sudo` command, you can check our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
+
+## Installing WP-CLI
+
+1.  WP-CLI is available as a PHP Archive file(phar). You can download it using either wget or curl commands.
+
+        curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+
+    Or
+
+        wget https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
+
+2.  You need to make this .phar file executable and move it to `/usr/local/bin` so that it can be run directly.
+
+        chmod +x wp-cli.phar
+        sudo mv wp-cli.phar /usr/local/bin/wp
+
+3.  Check if it is installed properly.
+
+        wp --info
+
+    You should see a similar output like the one below which means we can move ahead.
+
+        PHP binary:	/usr/bin/php5
+        PHP version:	5.6.11-1ubuntu3.1
+        php.ini used:	/etc/php5/cli/php.ini
+        WP-CLI root dir:	phar://wp-cli.phar
+        WP-CLI global config:
+        WP-CLI project config:
+        WP-CLI version:	0.21.1
+
+You can use the above procedure for upgrading WP-CLI as well.
+
+### Activating Bash-Completion
+
+Now it is difficult all commands at once and you might want a quick reference on the command you need. While you can always type `wp` and press enter to see the complete list. But wouldn't it be better if you can point to specific command. Like if you want to know what all sub commands are under the `core` parameter. Bash completion script of WP-CLI can help here.
+
+1.  To install the script, make sure you are in the home directory of the user(`username` in this case) you are using.
+
+        cd /home/username
+        wget https://github.com/wp-cli/wp-cli/raw/master/utils/wp-completion.bash
+
+2.  Now edit the .bashrc so that it is loaded by the shell every time you login. Open the file and add the following line in the editor assuming you downloaded the file in the home directory.
+
+        {: .file-excerpt}
+        /home/username/.bashrc
+        :   ~~~ bash
+        source /home/username/wp-completion.bash
+        ~~~
+
+3.  Now run the following command to reload the Bash profile.
+
+        source ~/.bashrc
+
+That's it. Bash Completion is enabled.
+
+## Basics of WP-CLI
+Before we get into the helm of the things, let us learn some essential basics of how exactly WP-CLI works. This would help you feel comfortable with what is going to follow next.
```

Comment:
I've never before heard or read the term "into the helm of things". Can you rephrase to something last colloquial?